apps/api/src/coscrad-cli/data-dump.cli-command.ts

```diff
@@ -11,12 +11,12 @@ import { CliCommand, CliCommandOption, CliCommandRunner } from './cli-command.de
     description: 'dumps the database state to a snapshot file',
 })
 export class DomainDumpCliCommand extends CliCommandRunner {
-    private readonly dataExporter: DataExporter;
+    private readonly dataExporter: DomainDataExporter;
 
     constructor(@Inject(REPOSITORY_PROVIDER_TOKEN) repositoryProvider: IRepositoryProvider) {
         super();
 
-        this.dataExporter = new DataExporter(repositoryProvider);
+        this.dataExporter = new DomainDataExporter(repositoryProvider);
```

Comment:
We need to update this command to leverage the Arango data exporter instead.

Reply:
[Here](https://www.pivotaltracker.com/story/show/185207104) is a story for that.